demos/instr.h

```diff
@@ -70,6 +70,19 @@ inline void MemoryAndSpeculationBarrier() {
 #endif
 }
 
+SAFESIDE_ALWAYS_INLINE
+inline void GenerateNop() {
+#if SAFESIDE_MSVC
+  __nop();
+#elif SAFESIDE_GNUC
+  // Memory clobber is to prevent ICC to push ESI on the stack.
```

Comment:
nit: Memory clobber is to prevent ICC **from** **pushing** ESI **onto** the stack.

Reply:
Done.